package.json

```diff
@@ -12,6 +12,15 @@
   },
   "dependencies": {
     "com.unity.nuget.newtonsoft-json": "3.1.0",
-    "com.unity.textmeshpro": "3.0.6"
-  }
+    "com.unity.textmeshpro": "3.0.6",
+    "com.unity.inputsystem": "1.7.0",
+    "com.unity.cinemachine": "2.9.7"
+  },
+  "samples": [
+    {
+      "displayName": "Third Person Demo (URP)",
+      "description": "This third person demo is a sample Unity game demoing Regression Game’s Unity SDK",
```

Comment:
Let's add to this to clarify that your project MUST be using URP for this demo scene to work, otherwise you should pick the demo with the correct render pipeline (which I realize don't exist yet).

Reply:
Fixed